src/main/resources/application.yml

```diff
@@ -1,5 +1,5 @@
 server:
-  port: 9000
+  port: 8080
```

Comment:
it is not necessary to add 8080 port explicitly to application.yml - it's 8080 by default. You may just remove it from configuration

Reply:
has been corrected